### PR TITLE
RFC 6265bis: Clarify same-site for reload navigations

### PR DESCRIPTION
**Pull request by chlily1**
Addressing the questions raised in issue #628

### CODE REVIEW

**Comment by chlily1** [comment]
@englehardt  @miketaylr @sbingler -- PTAL, thanks!

File: draft-ietf-httpbis-rfc6265bis.md [hunk 1]
```diff
@@ -962,7 +962,17 @@ Note: The port component of the origins is not considered.
 
 A request is "same-site" if its target's URI's origin is same-site with the
 request's client's "site for cookies" (which is an origin), or if the request
```
Comment (sbingler) [inline draft-ietf-httpbis-rfc6265bis.md:964]: This sentence is difficult to parse, I think removing the comma immediately after "(which is an origin)" should help.

**Comment by miketaylr** [comment]
LGTM, thanks!

**Comment by englehardt** [comment]
LGTM, modulo sbingler comment.

File: draft-ietf-httpbis-rfc6265bis.md (hunk 1 above)
Comment (chlily1) [reply draft-ietf-httpbis-rfc6265bis.md]: Done, removed comma.

**Comment by chlily1** [comment]
@mikewest PTAL, thanks!

**Comment by sbingler** [comment]
LGTM

**Comment by miketaylr** [comment]
@chlily1 can you also add a changelog entry? At that point, it's probably good to land given @englehardt's LGTM.

**Pull request merged**